src/intel-map.coffee

```diff
@@ -10,8 +10,6 @@
 # Commands:
 #   hubot intelmap for <search>
 #
-# Author:
-#   therealklanni
```

Comment:
I'm not even that worried about it. I'm not trying to be a prick or something, and I'm more than happy to give you your well-deserved credit. In fact the **only** reason my name was there is because that was the format that the hubot-scripts team originally defined for documenting scripts (maybe it's not even used for anything, I don't know). I think we both agree that author names don't really belong in open source code, but I was following the standard set forth.

Please submit a PR adding your name to the README as a (major) contributor, and I'll be very happy to merge it. Seriously, all your contributions have been greatly appreciated and you deserve some recognition, so please do create that PR.